src/types/decimal/coefficient.rs

```diff
@@ -285,4 +273,81 @@ mod coefficient_tests {
         assert_eq!(pos_zero, pos_zero);
         assert!(pos_zero.ion_eq(&pos_zero));
     }
+
+    #[test]
+    fn is_negative_zero() {
+        assert!(Coefficient::negative_zero().is_negative_zero());
+        assert!(!Coefficient::new(Sign::Positive, 0).is_negative_zero());
+        assert!(!Coefficient::new(Sign::Positive, 5).is_negative_zero());
+    }
+
+    #[test]
+    fn is_positive_zero() {
+        assert!(Coefficient::new(Sign::Positive, 0).is_positive_zero());
+        assert!(!Coefficient::new(Sign::Positive, 5).is_positive_zero());
+        assert!(!Coefficient::negative_zero().is_positive_zero());
+    }
+
+    #[test]
+    fn is_negative() {
+        assert!(Coefficient::negative_zero().is_negative());
+        assert!(Coefficient::new(Sign::Negative, 5).is_negative());
+        assert!(!Coefficient::new(Sign::Positive, 5).is_negative());
+    }
+
+    #[test]
+    fn sign() {
+        assert_eq!(Coefficient::negative_zero().sign(), Sign::Negative);
+        assert_eq!(Coefficient::new(Sign::Positive, 0).sign(), Sign::Positive);
+        assert_eq!(Coefficient::new(Sign::Negative, 5).sign(), Sign::Negative);
+        assert_eq!(Coefficient::new(Sign::Positive, 5).sign(), Sign::Positive);
+    }
+
+    #[test]
+    fn magnitude() {
+        assert_eq!(Coefficient::negative_zero().magnitude(), &UInt::from(0u32));
+        assert_eq!(
+            Coefficient::new(Sign::Positive, 0).magnitude(),
+            &UInt::from(0u32)
+        );
+        assert_eq!(
+            Coefficient::new(Sign::Negative, 5).magnitude(),
+            &UInt::from(5u32)
+        );
+        assert_eq!(
+            Coefficient::new(Sign::Positive, 5).magnitude(),
+            &UInt::from(5u32)
+        );
+    }
+
+    #[test]
+    fn convert_to_int() {
+        // i64
+        assert_eq!(
+            Int::try_from(Coefficient::new(Sign::Positive, 5)),
+            Ok(Int::from(5))
+        );
+        assert_eq!(
+            Int::try_from(Coefficient::new(Sign::Negative, 5)),
+            Ok(Int::from(-5))
+        );
+
+        // BigInt
+        let enormous_int = BigInt::from_str("1234567890123456789012345678901234567890").unwrap();
```

Comment:
Just for fun, this is a trick I sometimes use when making strings of numbers for length:
```suggestion
        let enormous_int = BigInt::from_str("1234567891023456789202345678930234567894").unwrap();
```
This allows you to more easily figure out how long the string is, find a `9x0` and the digit `x` is the 10s place not a zero. The following 1 is not valuable (inferable from `234..`), so I write `20`, `30`, what have you instead.